cmd/blockchaincmd/upgradecmd/apply.go

```diff
@@ -319,9 +318,6 @@ func applyPublicNetworkUpgrade(blockchainName, networkKey string, sc *models.Sid
 
 func validateUpgrade(blockchainName, networkKey string, sc *models.Sidecar, skipPrompting bool) ([]params.PrecompileUpgrade, string, error) {
 	// if there's no entry in the Sidecar, we assume there hasn't been a deploy yet
-	if sc.Networks[networkKey] == (models.NetworkData{}) {
-		return nil, "", subnetNotYetDeployed()
-	}
 	chainID := sc.Networks[networkKey].BlockchainID
```

Comment:
same here. should we check for existance of the key `networkKey`  in the `sc.Networks` map

Reply:
https://github.com/ava-labs/avalanche-cli/pull/2107#discussion_r1749295060